src/main/resources/application1.properties

```diff
@@ -1,4 +1,4 @@
 mongo.connectionString=mongodb+srv://alex:Alex_coder152@cluster0.wdxsl.mongodb.net/myFirstDatabase?retryWrites=true
 mongo.DBName=dv_spring_web_mongo
 spring.main.banner-mode=off
-server.port=1111
\ No newline at end of file
+logging.level.com.springweb.dv_spring_web_mongo.aspect:DEBUG
```

Comment:
I think that such property string cannot be parsed correcrtly. Please use '=' instead of ':'